tests/equilibria/test_flux_surfaces.py

```diff
@@ -40,6 +41,44 @@
 TEST_PATH = get_bluemira_path("equilibria/test_data", subfolder="tests")
 
 
+class TestFluxSurface:
+    def setup_class(self):
+        eq_name = "DN-DEMO_eqref.json"
+        filename = os.sep.join([TEST_PATH, eq_name])
+        self.eq = Equilibrium.from_eqdsk(filename)
+
+    def test_calculate_poloidal_angle(self):
+        # Flux surface in the SoL
+        x_start, z_start = 14, 0
```

Comment:
Are `x_start` and `z_start` integers ?

If thet are floats please use floats to define them so we avoid int to float convertion and other mess related to integer (more of a good practice than an issue for this specific code)

```suggestion
        x_start, z_start = 14.0, 0.0
```